config/discovery-db.container

```diff
@@ -5,7 +5,7 @@ PartOf=discovery-app.service
 
 [Container]
 ContainerName=discovery-db
-Image=registry.redhat.io/rhel8/postgresql-12:latest
+Image=registry.redhat.io/rhel8/postgresql-15:latest
```

Comment:
```suggestion
Image=registry.redhat.io/rhel9/postgresql-15:latest
```

Reply:
Ahh, should be good too 👍 . Let me try it.

Reply:
QE has been using `rhel9/postgresql-15` in their CI for a few weeks. I'm pretty sure it's stable. 🙂

Reply:
Just tested it with Quadlets @infinitewarp works fine. Just pushed an update.